docs/UserGuide.md

```diff
@@ -145,42 +145,75 @@ Switched to overview tab.
+It could be textbooks for the new semester, that cup of bubble tea you drank the other day, or even just taking public transport. 
+
+To help you manage your finances, Fine\$\$e records the following information about each of your expenses:
+1. **Title**: A descriptive title, to remind you what the expense was about.
+1. **Amount**: How much money you paid, to calculate how much you have spent in total.
+1. **Date**: The date the payment took place, to track your spending over periods of time.
+1. **Categories**: Any number of categories, to help you group related expenses together.
 
 #### 4.3.1 Add Expense
 
 Adds an expense to the finance tracker.
 
 Format: `add-expense t/TITLE a/AMOUNT d/DATE [c/CATEGORY]...`
 
-Shortcut: `adde t/TITLE a/AMOUNT d/DATE [c/CATEGORY]...`
+- `TITLE` should consist of <abbr title="Alphanumeric characters, space, and the special characters !&quot;#$%&'()*+,-./:;&lt;=&gt;?@[\]^_`{\|}~">printable ASCII characters</abbr>,
+and cannot begin with a space.
+- `AMOUNT` should be a number with 0 or 2 decimal places, with an optional `$` in front.
+- `DATE` should be in `dd/mm/yyyy` format, and cannot be later than the current date.
+- `CATEGORY` is optional. Multiple `c/` prefixes can be used to specify multiple categories.
+Each category name should consist of <abbr title="Alphanumeric characters, space, and the special characters !&quot;#$%&'()*+,-./:;&lt;=&gt;?@[\]^_`{\|}~">printable ASCII characters</abbr>,
+and cannot begin with a space.
 
-* `AMOUNT` should be a number with 0 or 2 decimal places, with an optional `$` in front.
-* `DATE` should be in `dd/mm/yyyy` format.
+Shortcut: `adde`, (when on the [Expenses tab](#)) `add`
 
 Examples:
-* `add-expense t/Bubble Tea a/5 d/03/10/2020 c/Food & Beverage` adds an expense with the title `Bubble Tea`, amount `$5.00`, date `03/10/2020` and category `Food & Beverage` to the finance tracker.
-* `adde t/Taxi home from school a/$13.50 d/10/10/2020 c/Transport c/School` adds an expense with the title `Taxi home from school`, amount `$13.50`, date `03/10/2020` and categories `Transport` and `School` to the finance tracker.
+- `add-expense t/Bubble Tea a/5 d/03/10/2020 c/Food & Beverage`
+adds a new expense titled `Bubble Tea`, with amount `$5.00`, date `03/10/2020`, and a single category `Food & Beverage`.
+- `adde t/Taxi Home from School a/$13.50 d/10/10/2020 c/Transport c/School`
+adds a new expense titled `Taxi Home from School`, with amount `$13.50`, date `10/10/2020`, and two categories `Transport` and `School`.
 
 Example Usage:
+```
+add-expense t/Bubble Tea a/5 d/03/10/2020 c/Food & Beverage
+```
 
 Expected Outcome:
+```
+New expense added: Bubble Tea Amount: $5.00 Date: 03/10/2020 Categories: [Food & Beverage]
+```
 
 #### 4.3.2 Edit Expense
 
-Edits an expense in the finance tracker.
+Edits an expense in the finance tracker. 
 
-Format:
+Format: (when on the [Expenses tab](#)) `edit INDEX [t/TITLE] [a/AMOUNT] [d/DATE] [c/CATEGORY]...`
```

Comment:
Link will be fixed when Tab section has been added.

![image](https://user-images.githubusercontent.com/53135010/97093515-14c45180-167f-11eb-8a9b-74b68ffb1604.png)